src/app/containers/MediaPlayer/helpers/embedUrl.js

```diff
@@ -0,0 +1,12 @@
+const embedUrl = (assetId, vpid, amp = false) => {
+  const baseUrl = process.env.SIMORGH_EMBED_BASE_URL;
```

Comment:
Let's start out with calling this `SIMORGH_AV_EMBED_BASE_URL` as before long we'll have other embeds (social etc.) that won't use the same endpoint.